packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -932,16 +942,18 @@ class DataGrid extends React.Component {
         isSelectable={cell.isSelectable}
         isSelected={cell.isSelected}
         selectableRefCallback={(ref) => { this.cellRefs[cellKey] = ref; }}
+        isColumnHighlighted={isColumnHighlighted}
```

Comment:
can we update this like below so that we won't need a variable declaration on L932.  
```suggestion
        isColumnHighlighted={(column.id === columnHighlightId)}
```

Reply:
[Updated](https://github.com/cerner/terra-clinical/pull/796/commits/d61cc802366b240b410fd07f004d522d32624e81#diff-412f754b40a1999c9f850f95d836fe5f21873db6a3feed84cbd813a65786e5ecR944)